BenadrylAutomation/src/test/java/com/jnj/testcases/FooterPageTest.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * @author Rashi Tiwari
+ * @date 18-March-24
+ */
+package com.jnj.testcases;
+
+import org.testng.SkipException;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import com.jnj.base.BaseClass;
+import com.jnj.dataproviders.FooterProvider;
+
+public class FooterPageTest extends BaseClass {
+	
+	@BeforeMethod
+	public void setup() throws InterruptedException {
+		launchApplication();
+	}
+	
+	@Test(priority = 1, description = "Verify the footer link of first column", dataProvider = "footer", dataProviderClass = FooterProvider.class, enabled = true)
```

Comment:
@RashiTiwari21 Please remove description for all test cases for which testcase name is coming from test data. This description is no longer needed.
Also, remove the enabled flag for all TC's since the execution is controlled from test data.

Reply:
fixed

Reply:
Verified @RashiTiwari21